bitvision/engine/data_bus.py

```diff
@@ -39,21 +39,25 @@ def page_config(tree):
     except:
         return {"title": "N/A", "date": "N/A"}
 
+
 def results_config(current_page):
     """
     Returns a config for Coindesk's search results page.
     """
 
     return {
         "page_url": ''.join(["https://www.coindesk.com/page/", str(current_page), "/?s=Bitcoin"]),
-        "item_XPATH": "//div[@class='post-info']", # XPATH for the search result item container
-        "url_XPATH": "./h3/a", # XPATH for url to full article, relative from item_XPATH
-        "date_on_page": True, # Whether it's possible to collect datetime objects from the results page
+        # XPATH for the search result item container
```

Comment:
The lines and comments should be formatted similarly. Either both come after the line or above it. The rest of our code is commented above the line, so I think that's the way to do it.

Reply:
Fixed.